### ISSUE TITLE
adding options to chnange brightness and change powerplans (classic)

### ISSUE THREAD

**Issue by ionuttbara** [michalleptuch/battery-flyout#4]
hi. brought recently your app and tested. i like that. but i miss some features from old flyouts.

**Comment by michalleptuch** [comment]
Hi, you can change display brightness using system action center.

I can add classic power plans support to my backlog.

**Comment by ionuttbara** [comment]
yes and no. i want to add the "Comfortable Backlight Control" from Battery Mode. 
https://en.bmode.tarcode.ru/
This functionality is used to control backlight on both internal and external monitors.
"Note: External monitors should support I2C interface and DDC/CI standard to work." (from Battery Mode website).

**Comment by michalleptuch** [comment]
I will push power plans in the next major update.

**Comment by michalleptuch** [comment]
Brightness slider and any related settings won't be added to this app. I will try to implement this functionality in Input Switcher.
